frontend/src/components/UserInfoCard.tsx

```diff
@@ -86,27 +86,27 @@ export const UserInfoCard = ({ userInfo }: { userInfo: UserInfo }) => {
   const emailOnOk = async (newValue: string) => {
     dispatch(
       modalsActions.showModal({
-        modalType: ModalTypes.CONFIRM_PASSWORD_MODAL,
+        modalType: ModalTypes.CHANGE_PASSWORD_MODAL,
```

Comment:
The change password modal doesn't make sense here.
Previously the modal was just used for asking the user to confirm the change by providing their password.

A separate modal would probably be good for this, since it feels separate from the functionality of changing a password.

Reply:
This is just me being careless. I didn't notice other functions used that modal and just refactored the word.

I need to be more careful about checking how my changes affect other functions in the future.